### DISCUSSION TITLE
does the library support autosuggest commands?

### DISCUSSION THREAD

**Discussion by anthamithya** [xetorthio/jedis#3123] [General]
Does Jedis supports ft.suggest, ft.suggest with payload commands?
I see that go library supports.

**Comment by sazzad16** [comment]
@anthamithya There is no FT.SUGGEST command.

I'm assuming you're meaning [FT.SUGGET](https://redis.io/commands/ft.sugget/) command. This feature is targeted for Jedis 4.3.0 and is already available in both 4.3.0-SNAPSHOT and `4.3.0-m1` release.

Also, payload is deprecated.

> Starting with Redis version 2.0.0: Deprecated WITHPAYLOADS argument

So Jedis is not including this support.

**Reply by realjays** [reply]
Thanks. Is there any alternative for for the payload? I would like to store a code associated for the suggestion.

**Reply by sazzad16** [reply]
@realjays You can use `T executeCommand(CommandObject<T>)` method to execute any command the way you want.

**Comment by anthamithya** [comment]
I mean, the payload itself is deprecated in SUGADD. So, any alternative way
to store additional details about the suggestion.

**Reply by sazzad16** [reply]
@anthamithya Please ask at [RediSearch](https://github.com/RediSearch/RediSearch).
